doc/sphinx/source/recipes/recipe_flato13ipcc.rst

```diff
@@ -549,6 +671,67 @@ Example plots
    shown. The error bars for the ESMs and observations represent interannual
    variability in the fluxes, calculated as the standard deviation of the
    annual means over the period 1986–2005.
+   
+.. figure:: /recipes/figures/flato13ipcc/fig-9-38.png
+   :align: center
+   
+   Figure 9.38: Mean seasonal cycle for the surface temperature (tas) 
+   as multi model mean of 38 CMIP 5 models and ERA-Interim data averaged 
+   for 1980-1999 over land in different regions: 
+   Western North America (WNA), Eastern North America (ENA), 
+   Central America (CAM), Tropical South America (TSA), 
+   Southern South America (SSA), Europe and Mediterranean (EUM), 
+   North Africa (NAF), Central Africa (CAF), South Africa (SAF), 
+   North Asia (NAS), Central Asia (CAS), East Asia (EAS), 
+   South Asia (SAS), Southeast Asia (SEA), and Australia (AUS).
+   
+   
+.. figure:: /recipes/figures/flato13ipcc/fig-9-38_regions.png
+   :align: center
+   
+   Figure 9.38reg: Positions of the regions used in Figure 9.38.
+   
+.. figure:: /recipes/figures/flato13ipcc/fig-9-39.png
+   :align: center
+   
+   Figure 9.39: Box and whisker plots showing the 5th, 25th, 50th, 75th
+   and 95th percentiles of the seasonal- and annual mean biases for 
+   the surface temperature (tas) between 34 CMIP5 models and ERA-Interim data. 
+   The regions are: Alaska/NW Canada (ALAs), 
+   Eastern Canada/Greenland/Iceland (CGIs), Western North America(WNAs), 
+   Central North America (CNAs), Eastern North America (ENAs), 
+   Central America/Mexico (CAMs), Amazon (AMZs), NE Brazil (NEBs), 
+   West Coast South America (WSAs), South-Eastern South America (SSAs), 
+   Northern Europe (NEUs), Central Europe (CEUs), 
+   Southern Europe/the Mediterranean (MEDs), Sahara (SAHs), 
+   Western Africa (WAFs), Eastern Africa (EAFs), Southern Africa (SAFs), 
+   Northern Asia (NASs), Western Asia (WASs), Central Asia (CASs), 
+   Tibetan Plateau (TIBs), Eastern Asia (EASs), Southern Asia (SASs), 
+   Southeast Asia (SEAs), Northern Australia (NASs) and 
+   Southern Australia/New Zealand (SAUs). 
+   The positions of these regions is defined following 
+   (Seneviratne et al., 2012) and differs from the ones in Fig. 9.38.
+   
+   
+.. figure:: /recipes/figures/flato13ipcc/fig-9-39_regions.png
+   :align: center
+   
+   Figure 9.39reg: Positions of the regions used in Figure 9.39.
+   
+
+.. figure:: /recipes/figures/flato13ipcc/fig-9-40.png
+   :align: center
+   
+   Figure 9.40: Box and whisker plots showing the 5th, 25th, 50th, 75th 
+   and 95th percentiles of the seasonal- and annual mean biases for 
+   the precipitation (pr) for oceanic and polar regions between 38 CMIP5 
+   models and CRU data for 1979–2000. 
+
+.. figure:: /recipes/figures/flato13ipcc/fig-9-41b.png
+   :align: center
+   
+   Figure 9.41b: Ranked modelled versus ERA-Interim mean temperature for
+   38 CMIP5 models in the Mediterranean region for 1979–2000.
```

Comment:
The recipe was set to produce the figure for two different regions. The number of regions results in the number of panels. I reduced it to the region shown in the IPCC and the example in the documentation to prevent confusion.